### DISCUSSION TITLE
请问作业那个prts.plus是不能用了吗

### DISCUSSION THREAD

**Discussion by WSMTBG** [MaaAssistantArknights/MaaAssistantArknights#3134] [Q&A] [answered]
https://prts.plus/
进去搜不到作业了
![微信截图_20221220185936](https://user-images.githubusercontent.com/32087250/208651513-45749939-fb8a-4217-8719-d67e1818f790.png)

**Comment by MistEO** [comment, marked as answer]
运维同学身体不适，等几天吧

**Reply by WSMTBG** [reply]
谢谢，大佬们注意防护
